Automattic/wp-calypso
# Marketplace Themes: Fixes the bug activating themes after resetting the site

## Proposed Changes

* Fixes the bug activating themes after resetting the site by requesting the marketplace products to be installed again.

## Testing Instructions



* Create a site and subscribe to Makoney
* **Reset** the site to remove all content using the atomic revert tool on MC (scroll to the bottom of the revert page and check 'I want to reset the site :siteSlug' 
* Navigate to `/themes/{YOUR_SITE_SLUG}`
* Find and activate Makoney
* The theme should be activated successfully

Closes #72444
